i18n/cs.json

```diff
@@ -465,7 +465,6 @@
         "backup": {
           "action": "Stáhnout zálohu...",
           "confirmationButton": "Stáhnout zálohu",
-          "confirmationText": "Prosím, zadejte své heslo pro stažení databázového souboru.",
```

Comment:
The delete is not necessary. Weblate should mark this as "to be adjusted" when source translation changes. Deleting it might introduce more conflicts.